tasks/run.py

```diff
@@ -0,0 +1,262 @@
+from invoke import task
+from multiprocessing import cpu_count
+from subprocess import run, PIPE, STDOUT
+from os.path import join, exists, dirname
+from tasks.util import (
+    RESULTS_DIR,
+    COVID_DIR,
+    NATIVE_BUILD_DIR,
+    FAASM_USER,
+    FAASM_FUNC,
+)
+from os import makedirs, remove
+from shutil import copyfile
+import requests
+import re
+
+COVID_SIM_EXE = join(NATIVE_BUILD_DIR, "src", "CovidSim")
+DATA_DIR = join(COVID_DIR, "data")
+
+FAASM_LOCAL_SHARED_STORE = "/usr/local/faasm/shared_store"
+FAASM_DATA_DIR = "faasm://covid"
+
+IMAGE_NAME = "experiment-covid"
+
+NATIVE_RESULTS_FILE = join(RESULTS_DIR, "covid_native.csv")
+WASM_RESULTS_FILE = join(RESULTS_DIR, "covid_wasm.csv")
+
+# Countries in order of size:
+# - Guam
+# - Malta
```

Comment:
@csegarragonz can you remember what the list is? I can't work out what order they go in (smallest/ quickest to largest/ slowest)

Reply:
It was something like (increasing):
* Guam
* US Virgin Islands
* Iceland
* Malta
* Alaska (quite longer)